client/tools/mgr-push/Makefile.python

```diff
@@ -2,18 +2,46 @@ THIS_MAKEFILE := $(realpath $(lastword $(MAKEFILE_LIST)))
 CURRENT_DIR := $(dir $(THIS_MAKEFILE))
 include $(CURRENT_DIR)../../../rel-eng/Makefile.python
 
+MGR-PUSH_ROOT_DIR = /manager/client/tools/mgr-push
```

Comment:
Are you sure MGR-PUSH_ROOT_DIR is valid for variable name?

Reply:
At least it seems to work, do you think it shouldn't for any reason?

Reply:
The `-` is sometimes a tricky character :) I would also have preferred using `MGRPUSH_ROOT_DIR` or `MGR_PUSH_ROOT_DIR`.

Reply:
Variable renamed :)